pkg/systemd/host.js

```diff
@@ -1356,15 +1354,17 @@ PageCpuStatus.prototype = {
     enter: function() {
         var self = this;
 
+        var n_cpus = 1;
+
         var options = {
             series: {shadowSize: 0,
                      lines: {lineWidth: 0, fill: true}
                     },
             yaxis: {min: 0,
-                    max: 100,
+                    max: n_cpus * 1000,
                     show: true,
                     ticks: 5,
-                    tickFormatter: function(v) { return (v / 10) + "%"; }},
+                    tickFormatter: function(v) { return (v / 10 / n_cpus) + "%"; }},
```

Comment:
This looks a bit confusing - so the y axis is multiplied by the number of CPUs, while the data is *divided* by the number of CPUs - that sounds duplicated?

Reply:
The data is not modified from what the metrics channel delivers, which is milliseconds of cpu usage per second.  Thus, with 4 cores, it can go up to 4000 ms/s.  The `tickFormatter` is only used to label the ticks, which will always go from 0 to 100% because we divide by 10 (to get from 1000 to 100) and by the number of cores (to get from 400 to 100).

This is different from all other plots, which do actually scale the actual data that goes into the plot.  This has always been the case, though.

Reply:
Thanks for the explanation!